dds/DCPS/RTPS/ICE/Checklist.h

```diff
@@ -161,6 +161,16 @@ struct Checklist : public Task {
 
   void check_invariants() const;
 
+  bool has_transaction_id(const STUN::TransactionId& a_tid) const
+  {
+    return std::find(connectivity_checks_.begin(), connectivity_checks_.end(), a_tid) != connectivity_checks_.end();
+  }
+
+  bool has_guid_pair(const GuidPair& a_guid_pair) const
+  {
+    return guids_.find(a_guid_pair) != guids_.end();
```

Comment:
Use `count()` here?  I guess it's equivalent.

Reply:
When returning a boolean, I tend to prefer a boolean expression.  It's longer, but in my opinion, easier to read.  When using `count` I have to make the extra mental step that non-zero returns as true.

Reply:
Starting in C++20 you can use `std::set::contains`.  Until then, I tend to use `count` since "does it have this key" (without needing an iterator) seems like the most fundamental operation on sets.  We don't need to change this code.